natgeo/mortar
# [AEM-6034] Making tweaks needed for AEM integration and bumping version

https://jira.natgeo.com/browse/AEM-6034

To test:

- Pull AEM-6034
- Run `npm test` to verify tests/linting pass
- Run `gulp serve`
- Navigate to http://localhost:3000/contentPackages/contentPackage/index.html
- Verify layout
